packages/components/src/composite/test/index.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import { queryByAttribute, render, screen } from '@testing-library/react';
+import { click, press, waitFor } from '@ariakit/test';
+import type { ComponentProps } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Composite } from '..';
+
+// This is necessary because of how Ariakit calculates page up and
+// page down. Without this, nothing has a height, and so paging up
+// and down doesn't behave as expected in tests.
```

Comment:
Ah, jsdom strikes again. Makes you wonder if this kind of tests isn't better suited for e2e. I guess not, because we're not really testing anything related to calculating the height.

Reply:
Not singularly for this test, but in general we way need to consider this. The closer our environment is to the real browser, the more confidence we get from the tests